front/src/pages/settings/SettingsProfile.tsx

```diff
@@ -44,6 +45,13 @@ export function SettingsProfile() {
             />
             <EmailField />
           </Section>
+          <Section>
```

Comment:
The blank space is too big:
<img width="422" alt="image" src="https://github.com/twentyhq/twenty/assets/12035771/395d046f-1d86-43c8-aae7-9ef0041b8725">

This is because both H2Title and Section are adding margin-bottom. I think the one on Section is right. The one on H2Title should be optional